### DISCUSSION TITLE
Getting started in Timetable Mode - guidance

### DISCUSSION THREAD

**Discussion by rickloader** [rickloader/NewForestRouteV3#9] [General]
The timetable is at the heart of New Forest Route (Nf) . Here is how the timetable is arranged ,and  timetable  to start the timetable
Trains on Nf are of 2 types - those with a locomotive, and STATIC trains without. Only Loco trains can be driven. 
The timetable is split into PLAYER - the trains that may be driven,  and STATIC - rolling stock with no loco and not drivable
1) From the OR Menu you choose a timetable set - PLAYER timetable
2) Choose a timetable. In the window some info shows the features of the selected timetable
3) Choose a train. The window shows part of the briefing to help you see  the task and decide if the train is of interest
![menuscreen](https://user-images.githubusercontent.com/57818551/184011527-f2da9ce2-4104-4678-b063-028e8ff8f25c.jpg)

**Comment by rickloader** [comment]
Problems with timetable selection occur if you forget to select the PLAYER timetable set
Going direct to the timetable (eg NfI3 currently) will load ok, but you will not have the static consists to couple to. AI traffic will also have trouble picking up non existant static consists.
If you select the STATIC timetable set (currentlyNfI1)  OR will object "Can`t find player locomotive" As a reminder the briefing window will say "Empty train Do not run"
I hope this clarifies. please feel free to comment
Rick

**Comment by rickloader** [comment]
Recent feedback ( thanks David) suggests that in the OR menu\options "Simple Controls and physics" should NOT be checked.
Enabling simple controls may cause consist faults when a saved game is resumed.

Always use a recent Unstable version of OR or the ORNYMG fork. For New Forest Route  purposes the stable versions of OR are out of date, because all route testing uses the Unstable version. Please do not be put off by the name , problems are rare in the unstable versions

**Comment by dforresc** [comment]
With reference to the first paragraph, I am finding the even if I ensure 
that "Simple Controls and physics" is unchecked, it will later re-check 
itself with disastrous results!

David
